### PR TITLE
Add stats: arena uptime.

### PR DESCRIPTION
**Pull request by interwq**

### CODE REVIEW

**Comment by interwq** [comment]
@jasone : I added the uptime for the merged arena stats, so that we still have it w/ shorter output (e.g. w/ "a" in malloc_stats_print). Also we won't need to special case the mallctl interface for MALLCTL_ARENAS_ALL.

File: doc/jemalloc.xml.in [hunk 1]
```diff
@@ -2280,6 +2280,19 @@ struct extent_hooks_s {
...
 
+      <varlistentry id="stats.arenas.i.uptime">
+        <term>
+          <mallctl>stats.arenas.&lt;i&gt;.uptime</mallctl>
+          (<type>uint64_t</type>)
+          <literal>r-</literal>
+        </term>
+        <listitem><para>Time elapsed (in nanoseconds) since the arena was
+        created.  If <mallctl>&lt;i&gt;</mallctl> equals
+        <mallctl>&lt;0&gt;</mallctl> or
```
Comment (jasone) [inline doc/jemalloc.xml.in]: Why support ```< 0```?

Comment (interwq) [inline doc/jemalloc.xml.in]: Oh this generates ```If <i> equals <0> or <MALLCTL_ARENAS_ALL>```. I'll get rid of the "<>" since it doesn't look necessary.

**Comment by jasone** [comment]
Oops, I meant to approve this yesterday.

**Pull request merged**